tests/js/environment.js

```diff
@@ -1,6 +1,6 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
```

Comment:
Most of the rest of the feedback is low-priority, but it would be great if this environment could be properly factored so that I can eliminate the huge cut and paste I currently have in Kettle tests at
https://github.com/amb26/kettle/blob/KETTLE-32/tests/shared/gpii-pouchdb-harness.js
The largest issue is just hoisting the "databases" up somewhere (and fixing the processing of "data" would save a tiny workaround)